demo/ThemeTransform.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang=en>
+
+  <head>
+    <meta charset=utf-8>
+    <title>ThemeTransform</title>
+    <style>
+      :root {
+        --control-panel-theme-visibility: visible;
+      }
+      body{
+        padding: 20px 20px 20px 300px !important;
+      }
+      .content_block {
+        padding-bottom: 50px;
+      }
+      #demoMenu a {
+        display: block;
+        color: black;
+      }
+      #demoTools {
+        background: #ccc;
+        padding: 15px;
+        position: fixed;
+        left: 20px;
+        top: 20px;
+        z-index: 10000;
+        width: 230px;
+        border-radius: 8px;
+      }
+      .demoArticleTitle {
+        color: red;
+        font-size: 2.0em;
+      }
+    </style>
+ 
+    <link href="https://en.wikipedia.org/w/load.php?debug=false&lang=en&only=styles&skin=vector&modules=skins.minerva.base.reset|skins.minerva.content.styles|ext.math.styles|" rel="stylesheet" type="text/css"></link>
```

Comment:
This ends with a closing pipe which seems incorrect. Android uses the following styles: `?debug=true&lang=en&modules=skins.minerva.base.reset|skins.minerva.content.styles|ext.cite.style|ext.math.styles|ext.timeline.styles|mediawiki.page.gallery.styles|mobile.app.pagestyles.android|mediawiki.skinning.content.parsoid&only=styles&version=&*`. I like to keep `debug=true` on for demos unless performance is bad. `mobile.app.pagestyles.android` is probably incorrect in the long term and we should probably update our usage to include more comprehensive styling from Minerva Neue, MediaWiki:*, and the extensions some day (but not today).

https://www.mediawiki.org/wiki/Manual:Interface/Stylesheets